From c4f49a77599dbe8867832f100304733f98e11c19 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Wed, 28 Dec 2011 17:41:58 +0800
Subject: [PATCH] Revert "rk29: ddr3sdk: defconfig: Use the CPUFreq governor
 "ondemand" as default"

This reverts commit 9e6a4573ac2d43e6198eeda26f9cbe1b66ca3250.

interactive governor save more power than ondemand.
---
 arch/arm/configs/rk29_ddr3sdk_defconfig | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/configs/rk29_ddr3sdk_defconfig b/arch/arm/configs/rk29_ddr3sdk_defconfig
index aed626a35667..beac7553e360 100755
--- a/arch/arm/configs/rk29_ddr3sdk_defconfig
+++ b/arch/arm/configs/rk29_ddr3sdk_defconfig
@@ -38,10 +38,11 @@ CONFIG_COMPACTION=y
 CONFIG_DEFAULT_MMAP_MIN_ADDR=32768
 CONFIG_CPU_FREQ=y
 CONFIG_CPU_FREQ_STAT_DETAILS=y
-CONFIG_CPU_FREQ_DEFAULT_GOV_ONDEMAND=y
+CONFIG_CPU_FREQ_DEFAULT_GOV_INTERACTIVE=y
+CONFIG_CPU_FREQ_GOV_PERFORMANCE=y
 CONFIG_CPU_FREQ_GOV_POWERSAVE=y
 CONFIG_CPU_FREQ_GOV_USERSPACE=y
-CONFIG_CPU_FREQ_GOV_INTERACTIVE=y
+CONFIG_CPU_FREQ_GOV_ONDEMAND=y
 CONFIG_CPU_FREQ_GOV_CONSERVATIVE=y
 CONFIG_VFP=y
 CONFIG_NEON=y
-- 
2.35.3

